src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -145,7 +156,30 @@ public static ValidateAIF create(ImmutableSet<CharSource> domainOntologySources,
             throw new IllegalArgumentException("Must validate against at least one domain ontology.");
         }
 
-        final Model model = ModelFactory.createDefaultModel();
+        Model model;
+        if (diskModelCount > 0) {
+            try {
+                // Make a disk-based model
+                Path directory;
+                do {
+                    directory = Paths.get(DOMAIN_MODEL_PATH + diskModelCount++);
+                    if (Files.exists(directory)) {
+                        Files.walk(directory).sorted(Comparator.reverseOrder()).map(Path::toFile).forEach(File::delete);
+                    }
+                }
+                while (Files.exists(directory)); // if it was in use, it wasn't deleted, which is a good thing.
+                Files.createDirectories(directory);
+                directory.toFile().deleteOnExit(); // not guaranteed
+                final Dataset dataset = TDBFactory.createDataset(directory.toString());
+                model = dataset.getDefaultModel();
```

Comment:
Hmm, it's been long enough that I'm not _really_ sure.  It probably had something to do with the fact that we're creating a union out of it and the data model, so maybe I thought they should both be backed by TDB so as not to "mix metaphors," so to speak.  But I don't recall if I read something that made me want to avoid that, or if it was just a "better safe than sorry" decision.

Thoughts?

Reply:
If we can avoid a disk model for the domain information, it would clean up the validateAIF code a bit and we'd only have to do the disk management in the CLI. I think the Union might build off whichever model is provided first (L). I'm running some tests with the dataToBeValidated as the first model.

Reply:
If we can avoid a disk model for the domain information, I think we don't need any changes to `ValidateAIF` for this PR at all.  If we know that the domain model doesn't take much room in memory (to justify making it disk-based) and that this doesn't cause any problems/inefficiencies with the union, then I'm all for keeping `ValidateAIF` untouched.

Reply:
Okay I've done enough tests with a memory-based domainModel that I've pushed an update where the domainModel is no longer based on TDB.  This also makes it smoother to run multiple validations at a time (no scary warnings).  I did change the order of the arguments in the call to `createUnion()` based on the documentation and a few tests.  Please do any final checks/tests you see fit, but I believe this was the final outstanding issue.